client/src/components/NewsBlock/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import hover_arrow from '../../assets/img/news-hover-arrow.svg'
+import { Link } from 'react-router-dom'
+
+export default function NewsBlock({ newsBlock }) {
```

Comment:
Why not const and arrow function without return?

Reply:
reworked